dev/cognitiveservices/data-plane/Language/questionanswering-authoring.json

```diff
@@ -33,7 +33,7 @@
     }
   ],
   "x-ms-parameterized-host": {
-    "hostTemplate": "{Endpoint}/language/authoring",
```

Comment:
@kayousef can we know the reason behind this ?. I believe the validation error due to this was overridden in preview October preview APIS.

Reply:
@heaths can you please response @sush2218 since @kayousef is OOF?

Reply:
Not sure how it was overridden, but the verification error came back. I told them to normalize the host template across swaggers to silence the validation, which seems bogus anyway since the end result is the same.